### PR TITLE
less aggressive success indication for adding an attachment

### PR DESCRIPTION
**Pull request by opqdonut**
instead of a flash message, show a green check mark next to the
attachment button

for #2616

# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [x] Link to issue
- [ ] Consider adding screenshots for ease of review

## Documentation
- [x] Update changelog if necessary
  - covered by existing entry
- [x] Components are added to guide page

## Testing
- [x] Valuable features are integration / browser / acceptance tested automatically
  - attachments covered by browser tests

### CODE REVIEW

**Comment by opqdonut** [comment]
screenshots for commit 4163ca3b37415f7ea6d21aa92a87a4def8f04aed

## before

success-before

## after 

success-after

**Comment by opqdonut** [comment]
screenshot of the feature itself, from the guide. it's best experienced interactively though!

indicator

File: src/cljs/rems/application.cljs [hunk 1]
```diff
@@ -168,17 +169,20 @@
...
      (save-application! description
                         application
                         (:field-values edit-application)
                         #(do
-                           (flash-message/show-default-success! :actions description)
-                           (rf/dispatch [::fetch-application (:application/id application)]))))
+                           (rf/dispatch [::fetch-application (:application/id application)])
+                           (if on-success
+                             (on-success)
+                             (flash-message/show-default-success! :actions description)))))
```
Comment (marharyta) [inline src/cljs/rems/application.cljs:185]: so you did decide to include this into your solution? nice!

**Comment by marharyta** [comment]
I really like the green checkboxes! Nice touch!

**Review by marharyta — APPROVED** [review]

**Comment by opqdonut** [comment]
success indicator removed as requested by @jaakkocsc

**Pull request merged**